Blockzilla/AppShortcuts/ShortcutView.swift

```diff
@@ -100,6 +100,16 @@ class ShortcutView: UIView {
         delegate?.shortcutTapped(shortcut: shortcut)
     }
     
+    func getShortcut() -> Shortcut {
+        return shortcut
```

Comment:
Make `var shortcut` `private(set)` and you will not need additional method to expose it.